programming/sed-article.md

```diff
@@ -178,25 +165,26 @@ contains the consecutive letters "abc".
 
 ### Special Characters and `sed`
 
-In the sections ahead we will see that some characters
+In the sections ahead, we will see that some characters
```

Comment:
You and Charles Sharp have opposite advice when it comes to commas. I used to use more, but he convinced me to use less. Apparently they are optional in many cases and it becomes a matter of preference.

Reply:
I'll tell Ms. Messik.

Reply:
Give her Charles' email address.